CONTRIBUTING.md

```diff
@@ -1,5 +1,16 @@
 # Contribution guidelines
 
+## Startup phase
+
+The RSQKit is still under heavy development.
+This means structure and content may still change a lot.
+During this phase, we organize contributions through EVERSE internal documents and communication channels.
+At some point we will likely move towards a more GitHub-centered contribution workflow, as outlined below.
+
+Specifically, the main "location" for assigning authors to contributions is the spreadsheet.
+Authors are to self organize (e.g. through Mattermost) in order to do collaborative writing.
+At the end, a single PR is submitted.
```

Comment:
Not sure what "single" means in this context.

Reply:
Ah, the discussion here was in the context of two people independently writing something similar and then making two "competing" PRs. This we want to prevent by self organizing via the spreadsheet. Then the end result is a "single PR" as opposed to two or more PRs :) But I agree that out of context "single" may not make much sense.

Reply:
Changed this now, please check if this is clearer.